moped-editor/src/views/projects/projectView/ProjectActivityLogDialog.js

```diff
@@ -98,10 +100,10 @@ const ProjectActivityLogDialog = ({ activity_id, handleClose }) => {
   };
 
   const getDateTime = data => {
-    const date = data?.moped_activity_log[0]?.created_at;
+    const date = adjustedTimezoneDateTime(data?.moped_activity_log[0]?.created_at); // adjustedTimezoneDateTime(data?.moped_activity_log[0]?.created_at);
```

Comment:
This line is doing weird syntax highlighting so its not totally clear whats happening. But it seems like we can delete the comment right?